### ISSUE TITLE
Error Running Program: TypeError: Cannot read property '1' of null

### ISSUE THREAD

**Issue by jeromebridge** [shannah/jdeploy#2]
I get the following stacktrace when I tried to run my installed java program.

````
home/vagrant/git/osgi-deployer/jdeploy-bundle/jdeploy.js:16
    var parts = match[1].split('.');
                     ^

TypeError: Cannot read property '1' of null
    at getJavaVersion (/home/vagrant/git/osgi-deployer/jdeploy-bundle/jdeploy.js:16:22)
    at Object.<anonymous> (/home/vagrant/git/osgi-deployer/jdeploy-bundle/jdeploy.js:98:19)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Function.Module.runMain (module.js:447:10)
    at startup (node.js:148:18)
    at node.js:405:3
````
This is the output of my `java -version` command:
````
vagrant@ubuntu:~/git/osgi-deployer$ java -version
openjdk version "1.8.0_111"
OpenJDK Runtime Environment (build 1.8.0_111-8u111-b14-2ubuntu0.16.04.2-b14)
OpenJDK 64-Bit Server VM (build 25.111-b14, mixed mode)
````

**Comment by shannah** [comment]
Thanks.  This should be fixed now when you update jdeploy to 1.0.19.  (npm update -g jdeploy)

**Comment by jeromebridge** [comment]
Thanks @shannah
